### DISCUSSION TITLE
Bubble Sort

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#19] [Q&A] [answered]
What is Bubble Sort? Answer Briefly.

**Comment by Preetiraj3697** [comment, marked as answer]
Bubble sort is a basic algorithm for arranging a string of numbers or other elements in the correct order. The method works by examining each set of adjacent elements in the string, from left to right, switching their positions if they are out of order. 
Implementing Bubble Sort Algorithm
Following are the steps involved in bubble sort(for sorting a given array in ascending order):

1.Starting with the first element(index = 0), compare the current element with the next element of the array.
2.If the current element is greater than the next element of the array, swap them.
3.If the current element is less than the next element, move to the next element. Repeat Step 1.

**Reply by ankit-akash** [reply]
congrats!🎉 you have got the Galaxy Brain badge.
